### PR TITLE
Action/create saves to knowledge

### PR DESCRIPTION
**Pull request by JosiahRegencia**
This is for issue #53

### CODE REVIEW

**Comment by JosiahRegencia** [comment]
Made a demo to show you how it works. Thought it would help a lot more from reading the code and visualizing how it works. Here's the link: https://www.youtube.com/watch?v=gQvFME4q5vw&feature=youtu.be

File: savant/app/results.py [hunk 1]
```diff
@@ -36,8 +36,8 @@ def compare():
 
     return new_packages
 
-@app.route('/doveps/api/action/create/<json_file>/<name>/<resource>/<action>', methods=['GET'])
```
Comment (greenmoss) [inline savant/app/results.py:39]: You still need to accept these parameters; see #49

File: savant/app/query.py [hunk 2]
```diff
@@ -66,20 +66,33 @@ def record_comparison_flavors():
 
 def record_knowledge():
     set_engine_name()
-    
-    if resource == 'Debs':
```
Comment (greenmoss) [inline savant/app/query.py:70]: The parameters from #49 still need to be used here.

**Review by greenmoss — CHANGES_REQUESTED** [review]

File: savant/app/results.py (hunk 1 above)
Comment (greenmoss) [reply savant/app/results.py:39]: When you used Vagrant to install apache2, it ran "apt-get install apache2". So for knowledge for this API request:
* name = apache2
* resource = Debs
* action = New

Comment (greenmoss) [reply savant/app/results.py:39]: So in the table, you need multiple rows:
* name "apache2", resource "package", action "install", packages table id "1" (for example)
* name "apache2", resource "package", action "install", packages table id "2" (for example)
* name "apache2", resource "package", action "install", users table id "1" (for example)

**Comment by greenmoss** [comment]
This is replaced by #68

**Pull request closed without merging**